docker-compose.static.yml

```diff
@@ -0,0 +1,41 @@
+# Container does not perform any initialization aside from importing env-vars from `confd`.
+# Users are expected to manually set up their site using a combination of the following:
+# - Makefile targets
+# - composer requires / install
+# - Drush commands
+# - Manual changes to the codebase directory
+version: "3.7"
+networks:
+  default:
+    internal: true
+  gateway:
+    external:
+      name: gateway
+volumes:
+  drupal-sites-data:
+  solr-data:
+services:
+  drupal:
+    build:
+      context: .
+      args:
+        TAG: ${TAG}
+        REPOSITORY: ${REPOSITORY}
+    image: ${REPOSITORY}/drupal-static:${DRUPAL_STATIC_TAG}
+    environment:
+      DRUPAL_INSTANCE: static
+    volumes:
+      - drupal-sites-data:/var/www/drupal/web/sites/default/files
+      - solr-data:/opt/solr/server/solr
+    depends_on:
+      # Requires a the very minimum a database.
+      - ${DRUPAL_DATABASE_SERVICE}
+    secrets:
```

Comment:
This file should probably define a top-level `secrets:` block so it doesn't depend on them being defined in another file.  That is, if a compose file relies on a `secret` it should define as a top-level key for the secret.  `docker-compose config` will merge duplicates.

Reply:
Ah, good catch